commands/inventory/add_service_mysql.go

```diff
@@ -96,14 +96,17 @@ func (cmd *addServiceMySQLCommand) Run() (commands.Result, error) {
 
 func (cmd *addServiceMySQLCommand) validateParams() error {
 	if cmd.Socket != "" {
-		if cmd.Address != "" || cmd.Port != 0 {
-			return fmt.Errorf("both socket and address are passed")
+		if cmd.Address != "" {
+			return fmt.Errorf("both socket and address cannot be specified")
```

Comment:
How about `socket and address cannot be specified together`? Because `both ...` sounds like no one of them could be specified

Reply:
sounds good